common-services-api/pom.xml

```diff
@@ -62,14 +67,14 @@
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
-                        <Bundle-SymbolicName>pentaho-hadoop-driver-V1</Bundle-SymbolicName>
-                        <Bundle-Version>9.9.9.9</Bundle-Version>
-                        <Pentaho-Code-Version>${pentaho-code.version}</Pentaho-Code-Version>
                         <DynamicImport-Package>*</DynamicImport-Package>
                         <Import-Package>
                         </Import-Package>
                         <_exportcontents>
                         </_exportcontents>
+                        <Export-Package>
+                            org.pentaho.big.data.api.jdbc.impl
```

Comment:
If you want to export the package do not use the `impl` (or `internal`) package name as these are conventioned to be implementation details that should not be exported https://github.com/pentaho/pentaho-coding-standards/wiki/3.3.-OSGi-best-practices.

Reply:
removed

Reply:
@pentaho/moseisley  @angel-ramoscardona you are still exporting classes from an `impl` package. You just substituted `impl` by `*`. 
https://github.com/pentaho/pentaho-hadoop-shims/tree/master/common-services-api/src/main/java/org/pentaho/big/data/api/jdbc/impl

If by design you want those classes to be exported, meaning that they are not implementation / internal details of the bundle as they are exposed to others, then they should be moved to another package. For example, moving them to the `org.pentaho.big.data.api.jdbc`package.

I didn't check the other `impl` packages. But if you are also exporting those somewhere, then they should also change their package.